build.zig

```diff
@@ -3,6 +3,16 @@ const Builder = @import("std").build.Builder;
 pub fn build(b: *Builder) void {
     const target = b.standardTargetOptions(.{});
     const optimize = b.standardOptimizeOption(.{});
+    // This is the library that can be imported by other programs
```

Comment:
This library does not expose a C API, so we don't build a static library and install it. The code that enables importing of this zig code in other zig code is the code below, which adds an `ansi-term` module.

Reply:
Thanks for the catch!